.travis.yml

```diff
@@ -11,6 +11,7 @@ env:
   - PIP=8.1.1
   - PIP=9.0.1
   - PIP=9.0.3
+  - PIP=10.0.1
   - PIP=master
```

Comment:
Also `PIP=18.0` should be tested. The master branch is not equal to the 18.0 version anymore, e.g. currently in master the `Command` class lives in `pip._internal.cli.base_command`, but in Pip 18.0 it's still in `pip._internal.basecommand`.

Reply:
Latest is tested, which is 18.  It doesn't particularly matter because the import shims are rewritten to handle both possibilities gracefully

Reply:
You're right, I forgot that latest==18 (currently).